src/test/apps.test.js

```diff
@@ -115,22 +115,65 @@ describe("Apps", function () {
       AppsActions.requestApps();
     });
 
-    describe("App", function () {
+    describe("basic app", function () {
```

Comment:
Sure, let's definitely go round and remove this at some point in the future though!